README.md

```diff
@@ -2,81 +2,28 @@ This is the code and configuration to carry out the antivirus checks on a single
 
 ## Building the lambda function
 
-There will be a task to automatically build the virus definitions into the lambda but for now it's a manual process.
-First, set $MANAGEMENT_ACCOUNT, STAGE and $YARA_VERSION. Current version as I write this is 4.0.0
+The lambda function is built by Jenkins. There are four Jenkins jobs in four Jenkinsfiles.
 
-```
-export MANAGEMENT_ACCOUNT=managementaccount
-export YARA_VERSION=4.0.0
-export STAGE=intg
-```
-
-Build the base yara image
-`docker build -f Dockerfile-yara --build-arg YARA_VERSION=$YARA_VERSION -t $MANAGEMENT_ACCOUNT.dkr.ecr.eu-west-2.amazonaws.com/yara:$STAGE .`
-
-Build the dependencies image
-
-`docker build -f Dockerfile-dependencies --build-arg YARA_VERSION=$YARA_VERSION -t $MANAGEMENT_ACCOUNT.dkr.ecr.eu-west-2.amazonaws.com/yara-dependencies:$STAGE .`
-
-Build the rules file container
-
-`docker build -f Dockerfile-compile -t $MANAGEMENT_ACCOUNT.dkr.ecr.eu-west-2.amazonaws.com/yara-rules --build-arg STAGE=$STAGE --build-arg ACCOUNT_NUMBER=$MANAGEMENT_ACCOUNT .`
-
-Run the dependencies container
-
-`docker run -itd --rm --name dependencies $MANAGEMENT_ACCOUNT.dkr.ecr.eu-west-2.amazonaws.com/yara-dependencies:$STAGE`
-
-Copy the dependencies zip locally
-
-`docker cp dependencies:/lambda/dependencies.zip .`
-
-Run the rules container
-
-`docker run -itd --rm --name rules yara-rules`
-
-Make a lambda directory
-
-`mkdir lambda`
+### Jenkinsfile-build
+There are three docker images that are used to build the lambda. 
 
-Copy the rules file into the lambda directory
+| File name               | Image Name        | Description                                                                                                               |
+|-------------------------|-------------------|---------------------------------------------------------------------------------------------------------------------------|
+| Dockerfile-yara         | yara              | Installs yara and some dependencies it needs like openssl on an alpine image.                                             |
+| Dockerfile-compile      | yara-rules        | Uses yara as the base image. Gets the yara rules from github and compiles them into a single file for yara to use         |
+| Dockerfile-dependencies | yara-dependencies | Installs necessary software on an amazon linux image and zips it up to be used by the lambda                              |
 
-`docker cp rules:/rules/output ./lambda`
+The build job rebuilds all of these images. This isn't necessary most of the time because the dependency versions and yara version don't change that often and so this job is only ever run manually when we need to update the dependencies.
```

Comment:
Thanks for adding these explanations! Could you add some deployment instructions as well?

e.g. "To deploy everything, run https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Antivirus%20Build/. If you just need to deploy changes to the lambda, run https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Antivirus%20Deploy/. (Assuming that's true? And you don't need to run any of the intermediate steps to do the tagging?)

Reply:
Good point, I've added a section on deploying the images. I've also added in the `--no-cache` and `--pull` arguments to the build commands so we get the latest versions every time.